docs/api/advanced/subscribe.mdx

```diff
@@ -35,6 +35,12 @@ subscribe(state.arr, () => console.log('state.arr has changed to', state.arr))
 state.arr.push('world')
 ```
 
-## Codesandbox demo in VanillaJS
+#### no usage tracking
+
+`subscribe` currently does not implement the fine-grained usage tracking of `useSnapshot`, so the callback will rerun anytime the subscribed proxy (or one of its child proxy) has any fields mutated, regardless of the fields accessed within the `callback`'s code.
```

Comment:
such that it will _never_ be implemented.

Reply:
Ah yeah, I think I see what you mean: per that other paragraph I added, I understand why valtio's core `addListener` + subscription primitives must never be usage-based, b/c snapshots must be re-created on any field change.

That said, when I wrote this, I was thinking that technically `subscribe`-the-exported-method _could_ learn to do usage tracking, w/o changing the other internals...

Granted, the API would change b/c the callback would need to accept a snapshot.

But, maybe at that point `subscribe` would be too much like a `watch`-that-learned-to-do-usage-tracking...

So, yeah, that makes sense; I'll edit this to remove the insinuation that `subscribe`-the-public-api could be usage based someday.

Reply:
Okay, I rewrote this denote subscribe will always stay non-usage-based, b/c its primary purpose is to drive snapshot creation.